virtcontainers/pkg/nsenter/nsenter_test.go

```diff
@@ -138,30 +138,31 @@ func TestSetNSWrongFileFailure(t *testing.T) {
 	assert.NotNil(t, err, "Should fail because file is not a namespace")
 }
 
-var testNamespaceList = []Namespace{
-	{
-		Type: NSTypeCGroup,
-	},
-	{
-		Type: NSTypeIPC,
-	},
-	{
-		Type: NSTypeNet,
-	},
-	{
-		Type: NSTypePID,
-	},
-	{
-		Type: NSTypeUTS,
-	},
+func supportedNamespaces() []Namespace {
```

Comment:
This is duplicated with the init function, you can refactor and add unit test. Also suggest to add some log when a one namespace is not supported.

Reply:
my bad, it is not duplicated. Just the log would be great

Reply:
it's a package, we don't have logs